RELEASE.md

```diff
@@ -0,0 +1,92 @@
+
+1. **API refactors/changes**. There is equivalent functionality, but the API has changed. Generally adapting to these changes is trivial, and release notes will contain information about how to adapt.
+
+1. **Removal of a module or functionality**. In this case, Boxo maintainers will deprecate the relevant types in at least one release prior to their removal, and users will be forewarned in release notes. We strive to remove functionality as rarely as possible.
+
+We know that breaking changes may cause short-term pain, and we only make breaking changes when we believe the long-term gain is much greater than the short-term pain.
+
+Feedback from users helps us in this determination, so *please* provide feedback on any GitHub issues or PRs that may affect you, and let us know when we're wrong! We believe there's *a lot* of innovation and growth for IPFS in the future, so we don't want Boxo to ossify.
+
+### Versioning
+Boxo only releases minor and patch versions, and has no plans to change major versions from v0.
+
+Minor versions contain new or removed functionality, or significant changes in dependencies.
+
+Patch versions contain "fixes", which are generally small, targeted, and involve limited changes in functionality (such as bug fixes, security fixes, etc.). 
+
+### Backporting
+The amount of backporting of a fix depends on the severity of the issue and the impact on users. Most non-critical fixes won't be backported.
+
+As a result, Boxo maintainers recommend that consumers stay up-to-date with Boxo releases.
+
+### Go Compatibility
+At any given point, the Go team supports only the latest two versions of Go released (see https://go.dev/doc/devel/release). Boxo maintainers will strive to maintain compatibilty with the older of the two supported versions, so that Boxo is also compatible with the latest two versions of Go.
+
+### Release Criteria
+Boxo releases occur _at least_ on every Kubo release. Releases can also be initiated on-demand, regardless of Kubo's release cadence, whenever there are significant changes (new features, refactorings, deprecations, etc.).
+
+The [release process](#release-process) below describes how maintainers perform a release.
+
+Boxo may release on Fridays or even weekends. This is in contrast to Kubo, which does not release on Fridays since it is a deployable binary, to avoid weekend surprises.
+
+### Testing
+Boxo maintainers are in the process of moving tests into Boxo from Kubo (see [#275](https://github.com/ipfs/boxo/issues/275)) . Until that's done, Boxo releases rely on Kubo tests for extra confidence. So releases must plumb Boxo into Kubo and ensure its tests pass. See [Release Process](#release-process) for details.
+
+### Changelogs
+Boxo loosely follows the [Keep a Changelog](https://keepachangelog.com/en/1.0.0/) format. New commits should add changelog entries into the `[Unreleased]` section so that there is no changelog scramble when a new version needs to be released.
+
```

Comment:
```suggestion
Boxo loosely follows the [Keep a Changelog](https://keepachangelog.com/en/1.0.0/) format. New commits should add changelog entries into the `[Unreleased]` section so that there is no changelog scramble when a new version needs to be released.

We take to heart the "guiding principle" that "changelogs are for humans, not machines.".  As a result:
* Breaking changes and notable features are called out following the changelog emoji legend.
* Changelog entries provide links to a relevant issue/PR so an interested party can learn more.
```